gatsby-config.js

```diff
@@ -23,37 +24,51 @@ module.exports = {
     )
   },
   plugins: flatten([
-    process.env.GATSBY_ACTIVE_ENV !== 'docs' ? [
-      {
-        resolve: `gatsby-plugin-page-creator`,
-        options: {
-          path: `${__dirname}/content/pages`,
-        },
-      },
-      {
-        resolve: `gatsby-source-wordpress`,
-        options: {
-          baseUrl: 'sparkpost.com',
-          protocol: 'https',
-          hostingWPCOM: false,
-          useACF: false,
-          excludedRoutes: [ '**/oembed/**', '**/akismet/**', '**/yoast/**' , '**/users/**', '**/settings', '**/pages', '**/yst_prominent_words', '**/comments', '**/statuses', '**/media' ]
-        }
-      }
-    ] : [],
-    process.env.GATSBY_ACTIVE_ENV === 'publish' ? [
-      {
-        resolve: `gatsby-plugin-algolia`,
-        options: {
-          appId: 'SFXAWCYDV8',
-          apiKey: process.env.ALGOLIA_TOKEN,
-          queries: [
-            require('./algolia/apiReference'),
-            require('./algolia/momentum')
-          ]
-        }
-      }
-    ] : [],
+    process.env.GATSBY_ACTIVE_ENV !== 'docs'
+      ? [
+          {
+            resolve: `gatsby-plugin-page-creator`,
+            options: {
+              path: `${__dirname}/content/pages`,
+            },
+          },
+          {
+            resolve: `gatsby-source-wordpress`,
+            options: {
+              baseUrl: 'sparkpost.com',
+              includedRoutes: [
+                // "**/categories",
+                '**/posts',
+                // "**/pages",
+                // '**/media',
+                // "**/tags",
+                // "**/taxonomies",
+                '**/users',
+              ],
+              hostingWPCOM: false,
+              minimizeDeprecationNotice: false, // todo, we need to upgrade, please don't silence this notice
```

Comment:
Set `minimizeDeprecationNotice` with a comment, so we don't accidently silence this notice.  It is important that it remains on our radar to upgrade ASAP.